UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -57,6 +57,20 @@ public class LengthTests : LengthTestsBase
 
         protected override double HandsInOneMeter => 9.8425196850393701;
 
+        protected override double AstronomicalUnitsInOneMeter => 1.0 / 1.4959787070e11;
```

Comment:
These values should not be calculations. It's important that they are values from external references and easily verifiable, because it is very easy to get the from/to conversions the wrong direction.

Reply:
Problem is that the value that is well known in the litterature is the inverse of what is asked for, not the value "AU2Meter". I have no problem changing it though.

Reply:
Got it, well if you can't find the value in an online converter or elsewhere, then you can always compute the value. I will double check the numbers as part of reviewing the PR.

Reply:
Values are found in online calculator. But I like that we have the well known value as is in the code. But I don't want to mess with your code conventions. I will put the calculated value.